### PR TITLE
Remove receipt handling

### PR DESCRIPTION
**Pull request by gbanas**

### CODE REVIEW

File: app/response_processor.py [hunk 1]
```diff
@@ -36,8 +38,17 @@ def process(self, encrypted_survey):
...
             return False
 
-        receipt_ok = self.send_receipt(decrypted_json)
-        if not receipt_ok:
+        receipt_json = {
+            'tx_id': decrypted_json['tx_id'],
+            'collection': {'exercise_sid': decrypted_json['collection']['exercise_sid']},
+            'metadata': {
+                'ru_ref': decrypted_json['metadata']['ru_ref'],
+                'user_id': decrypted_json['metadata']['user_id']}
```
Comment (necrophonic) [inline app/response_processor.py]: Stylistically closing `}` should be in the next line

```
{
	'tx_id': decrypted_json['tx_id'],
	'collection': {
		'exercise_sid': decrypted_json['collection']['exercise_sid']
	},
	'metadata': {
		'ru_ref': decrypted_json['metadata']['ru_ref'],
		'user_id': decrypted_json['metadata']['user_id']
	}
}
```

File: app/response_processor.py [hunk 2]
```diff
@@ -1,17 +1,19 @@
-from app import receipt
 from app import settings
+from app.queue_publisher import QueuePublisher
 from app.settings import session
+from json import dumps
 from requests.packages.urllib3.exceptions import MaxRetryError
 
 
+def queue_receipt(logger, payload):
```
Comment (necrophonic) [inline app/response_processor.py]: Only used in one place so probably not worth factoring out to it's own function

File: app/response_processor.py [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
+    publisher = QueuePublisher(logger, settings.RABBIT_URLS, settings.RABBIT_RRM_RECEIPT_QUEUE)
```
Comment (necrophonic) [inline app/response_processor.py]: Do you need to instantiate every time you need to publish or can you reuse a publisher instance?

File: app/response_processor.py (hunk 2 above)
Comment (gbanas) [inline app/response_processor.py]: Fixed in: https://github.com/ONSdigital/sdx-collect/commit/43f34c7fe99a1e4fde1cd36558089d926aaed28b

File: app/response_processor.py (hunk 3 above)
Comment (gbanas) [inline app/response_processor.py]: Fixed in: https://github.com/ONSdigital/sdx-collect/commit/43f34c7fe99a1e4fde1cd36558089d926aaed28b

**Comment by necrophonic** [comment]
Will need to have the environment variables in build envs updated to match the new/changed vars when being deployed.

**Pull request merged**